### ISSUE TITLE
PrettyPrint needs a config argument so output does not re-escape backslashes. If I print var it has single \

### ISSUE THREAD

**Issue by JGFMK** [python/cpython#119411]
# Bug report

### Bug description:

```python
import pprint
parts = ['a','b']
n = '\\'.join(parts)
print(n)
outf = open('C:/temp/out.txt', 'w', encoding='utf-8')
PP = pprint.PrettyPrinter(indent=4,stream=outf,width=220)
PP.pprint(n)
outf.close()
```
out.txt will contain doubled up \ .. But when n is printed to the console, it's not escaped.
There needs to be a config option to disable this fubar side effect.

### CPython versions tested on:

3.12

### Operating systems tested on:

Windows

**Comment by serhiy-storchaka** [comment]
`print()` outputs the `str()` of the object. `pprint.pprint()` outputs the wrapped `repr()` of the object. They are fundamentaaly different. You can also notice that the pprint output contains quotes around string.

If you need to output the string literally, use `print()` (or just the file's `write()`  method). If you want to wrap the string, use the `textwrap` module. `pprint` has nothing to do with this.

**Comment by ericvsmith** [comment]
Agreed with @serhiy-storchaka. @JGFMK: what's your use case for wanting `pprint` to use `str`, not `repl`? Why not just call `str` yourself instead of using `pprint`?

**Comment by devdanzin** [comment]
@JGFMK do you have any feedback on this? Otherwise we shall close it as working-as-planned.
